### DISCUSSION TITLE
Question: Do functions work on namespace buckets?

### DISCUSSION THREAD

**Discussion by macthev** [noobaa/noobaa-core#7010] [General]
I have simple functions that gets called when creating objects on first.bucket however if I define a namespace bucket no such call is logged. Are functions supported on namespace buckets?

first.bucket
 [Endpoint/15]    [L0] core.server.func_services.func_server:: invoking on server helloworld

regards..

**Comment by macthev** [comment]
looks like this https://github.com/noobaa/noobaa-core/pull/6587 is my answer.

**Comment by liranmauda** [comment]
@macthev some flows are not working with lambda, and we are in the process of adding support to Knative and replacing the lambda functionality with it.

**Reply by macthev** [reply]
@liranmauda thanks
